tools/fluent/basic_syntax.md

```diff
@@ -113,19 +106,30 @@ emails = { $unreadEmails ->
 
 While in most cases messages will have a single value, it’s also possible to define multiple **variants**: different values, each one associated to a key, that can be selected when referencing the message ID from another message. Unlike selectors, the message doesn’t depend on the value of an external variable or function.
 
-This can be useful, for example, to decline names according to grammatical case.
+This can be useful, for example, to translate according to grammatical case or gender. The following example combines a private message (`-creature-elf`) and variants.
+
+In English:
+
+```PROPERTIES
+-creature-elf = an Elf
+
+you-found-elf = You found { -creature-elf }.
+elf-found-you = Oops, { -creature-elf } found you.
+```
+
+In German:
 
 ```PROPERTIES
--brand-name = {
-       *[nominative] Aurora
-        [genitive] Aurore
-        [dative] Aurori
-        [accusative] Auroro
-        [locative] Aurori
-        [instrumental] Auroro
+-creature-elf =
+    {
+       *[Nominativ] ein Elf
+        [Akkusativ] einen Elf
     }
 
-about = O { -brand-name[locative] }
+you-found-elf = Du hast { -creature-elf[Akkusativ] } gefunden.
+elf-found-you = Ups, { -creature-elf[Nominativ] } dich gefunden.
```

Comment:
This is missing the auxiliary verb: `hat dich gefunden` :)

Reply:
Ugh, poor German